src/ui/templates/results/pagination.hbs

```diff
@@ -38,11 +40,12 @@
     </a>
 
     <a
-      class="yxt-Pagination-link js-yxt-Pagination-link{{#unless ellipses.desktopBack}} desktop-hidden{{/unless}}{{#unless ellipses.mobileBack}} mobile-hidden{{/unless}}"
+      class="yxt-Pagination-link js-yxt-Pagination-link{{#unless pinnedNumbers.desktopBack}} desktop-hidden{{/unless}}{{#unless pinnedNumbers.mobileBack}} mobile-hidden{{/unless}}"
```

Comment:
Would it be simpler to just conditionally render these tags instead of toggling hidden CSS classes on or off?

Reply:
the limits are different values for mobile and desktop, and there's no way to conditionally render them from the handlebars while knowing the size of the browser (at least not without some extremely hacky js)